src/nvim/highlight.c

```diff
@@ -927,10 +927,11 @@ int object_to_color(Object val, char *key, bool rgb, Error *err)
   } else if (val.type == kObjectTypeString) {
     String str = val.data.string;
     // TODO(bfredl): be more fancy with "bg", "fg" etc
+    if (!str.size || STRCMP(str.data, "NONE") == 0) {
```

Comment:
Should we use `STRICMP` so that it's case insensitive? At least with `:highlight`, `NONE`, `none`, and `NoNe` all work.